shotgun_api3/shotgun.py

```diff
@@ -2374,11 +2355,17 @@ def _get_attachment_upload_info(self, is_thumbnail, filename):
         if not str(upload_info).startswith("1"):
             raise ShotgunError("Could not get upload_url but " \
                                "not sure why.\nPath: %s\nUrl: %s\nError: %s" % (
-                                   path, url, str(upload_info)))
+                                   filename, url, str(upload_info)))
 
         LOG.debug("Completed rpc call to %s" % (upload_url))
+
+        upload_info_parts = str(upload_info).split("\n")
```

Comment:
do we need to worry about different versions of the sg server responding in different ways here? or is it consistent - not sure if the server side changes are going out as a patch release or if they are perhaps already there. If they are going out in a patch, we might have to add checks to handle different server responses differently?